src/@types/types.d.ts

```diff
@@ -127,6 +127,7 @@ export type RouteDefinition = {
   manifestLocation: string;
   dynamic?: boolean;
   exact?: boolean;
+  props?: any;
```

Comment:
We want to consume these props on two places? In the route definition[1] and in fed-modules route config[2]? It would be better to choose one so app developers are not confused which props they should use.

[1] https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/rhel-navigation.json#L6
[2] https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/fed-modules.json#L10

Reply:
We should use the one just for routes. It's similar to how you would a component in a source code.